package.json

```diff
@@ -52,21 +52,21 @@
         "eslint:workspace": "npm run eslint:cmd -- '.*.*' '*.*' .vscode .github docker",
         "eslint:cmd": "npx eslint --ignore-path .eslintignore --no-error-on-unmatched-pattern --cache --cache-location node_modules/.cache/eslint/.eslintcache",
         "eslint:contracts": "npm -w @prosopo/captcha-contract -w @prosopo/common-contract -w @prosopo/proxy-contract run eslint",
-        "eslint:fix": "npm run ws -- eslint:fix && npm run eslint:fix:workspace",
+        "eslint:fix": "npx eslint --ignore-path .eslintignore --no-error-on-unmatched-pattern --cache --cache-location node_modules/.cache/eslint/.eslintcache",
```

Comment:
why has this changed?

Reply:
Because the linting is done as one command over the entire repository. It doesn't run in individual packages. We'd need to add extra ignore for when obf-bot is included.